common/txmgr/types/fee_estimator.go

```diff
@@ -14,9 +14,13 @@ const (
 	OptForceRefetch Opt = iota
 )
 
+type Fee interface {
+	String() string
+}
```

Comment:
Will this continue to evolve? Otherwise could remove some indirection by inlining `fmt.Stringer` instead.

Reply:
I believe this is only needed for logging, and don't expect this to evolve further.